metaflow/plugins/kubernetes/kubernetes_cli.py

```diff
@@ -167,6 +183,12 @@ def echo(msg, stream="stderr", job_id=None, **kwargs):
         kwargs["input_paths"] = "".join("${%s}" % s for s in split_vars.keys())
         env.update(split_vars)
 
+    if num_parallel is not None and num_parallel <= 1:
+        raise KubernetesException(
+            "Using @parallel with `num_parallel` <= 1 is not supported with Kubernetes. "
+            "Please set the value of `num_parallel` to be greater than 1."
+        )
+
```

Comment:
constraint added because jobset doesn't play nice with `replicas = 0`; Once `kubernetes-sigs/jobset` allow this, we can lift this constraint and add version logic to verify if the jobset can be submitted or not. Currently not supported with Jobset CRD version `jobset.x-k8s.io/v1alpha2`